src/signals/incident-management/containers/Dashboard/styled.ts

```diff
@@ -9,3 +9,19 @@ export const StyledRow = styled(Row)`
   background-color: ${themeColor('tint', 'level3')};
   margin: 0;
 `
+
+export const Title = styled.h6`
+  font-size: 18px;
+  line-height: 32px;
```

Comment:
this is 2 rem actually, 16 times 2, clearer imo

Reply:
outdated